string serial_id
Pose pose
---
Point[] path